e2e-identity/tests/e2e.rs

```diff
@@ -736,9 +751,10 @@ mod dpop_challenge {
         ));
     }
 
+    #[tokio::test]
     /// The DPoP token holds the "display name" of the client which is compared by the acme server against the
     /// display name in the acme identifier as part of the acme order
-    #[tokio::test]
+    // @SF.PROVISIONING @TSFI.ACME
```

Comment:
@emil-wire would it be possible to extend your comment parser so that this does not need to be the line just before the function?